modules/editTask.js

```diff
@@ -0,0 +1,20 @@
+export default function editTask(item, array) {
+  item.addEventListener('focusout', () => {
+    array.forEach((task) => {
+      if (task.index === item.id) {
+        task.description = item.value;
+        localStorage.setItem('array', JSON.stringify(array));
+      }
+    });
+  });
+  item.addEventListener('keyup', (e) => {
+    if (e.key === 'Enter') {
+      array.forEach((task) => {
+        if (task.index === item.id) {
+          task.description = item.value;
+          localStorage.setItem('array', JSON.stringify(array));
+        }
+      });
+    }
+  });
+}
```

Comment:
- [ ] Kindly ensure that after editing a task, the changes are saved in the local storage so that whenever a user refreshes a page, the changes persist

Reply:
Fixed.